Dockerfile.tools

```diff
@@ -17,18 +17,31 @@ RUN curl -L -s https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz > go1.13.8.l
     && mv go /usr/local \
     && rm -rf ./go*
 
+# Download and install oc
+RUN curl -L -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.2.2/openshift-client-linux-4.2.2.tar.gz -o openshift-origin-client-tools.tar.gz \
```

Comment:
Might want to use 4.4/4.5 for the client

Reply:
I prefer doing that in a separate PR. This is just a revert of what was wrongly removed in #192.